astro.config.mjs

```diff
@@ -45,8 +45,11 @@ export default defineConfig({
   integrations: [
     starlight({
       title: "Nemo Rule Engine",
```

Comment:
title should probably be `Nemo Graph Rule Engine`, in accordance with the logo?

Reply:
true, however it looks bad in the default design